src/components/AttachmentPicker/index.native.js

```diff
@@ -65,7 +66,7 @@ function getDataForUpload(fileData) {
         name: fileData.fileName || fileData.name || 'chat_attachment',
         type: fileData.type,
         uri: fileData.uri,
-        size: fileData.size,
+        size: lodashGet(fileData, 'fileSize', lodashGet(fileData, 'size')),
```

Comment:
Why do we need lodashGet here? will `FileData` ever be undefined?

Reply:
Still a question?

Reply:
Ohh I did answer this yesterday, can't find the comment.

Earlier we had `fileData.size` but it gives size is `undefined`. I checked it returns `fileData.fileSize`. So I've used both to keep it backward compatible.

Reply:
Ok. I think it is better like this.
```
Filedata.filesize || filedata.size
```

Reply:
Updated. I thought generally we use `lodash.get` for optional checks.